.github/workflows/unit-tests.yaml

```
@@ -0,0 +1,84 @@
+# yamllint disable rule:line-length
+# yamllint disable rule:braces
+
+name: Unit & Integration Tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - master
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.operating-system }}
+
+    strategy:
+      matrix:
+        operating-system: [ ubuntu-latest ]
+        php-version: [ '8.0' ]
+        coverage-driver: [ pcov, xdebug ]
+        include:
+          - { operating-system: 'windows-latest', php-version: '8.0', coverage-driver: 'xdebug' }
+          - { operating-system: 'ubuntu-latest', php-version: '8.0', coverage-driver: 'pcov' }
+          - { operating-system: 'ubuntu-latest', php-version: '8.0', coverage-driver: 'xdebug' }
+          - { operating-system: 'ubuntu-latest', php-version: '8.1', coverage-driver: 'pcov' }
+          - { operating-system: 'ubuntu-latest', php-version: '8.1', coverage-driver: 'pcov' }
```

```suggestion
          - { operating-system: 'ubuntu-latest', php-version: '8.1', coverage-driver: 'xdebug' }
```

because this is a duplicate of line 26